### PR TITLE
fix: support not defined contractType for NFT types.

### PR DESCRIPTION
**Pull request by b4rtaz**
---
name: [support not defined contractType for NFT types.](https://github.com/MoralisWeb3/Moralis-JS-SDK/commit/1ec4c2fe3003ab6a2e10a6ee5825145fd6d698ea)
---

### CODE REVIEW

File: packages/evmUtils/src/dataTypes/EvmNft/EvmNft.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export class EvmNft implements MoralisDataObject {
   static parse = (data: EvmNftInput, core: MoralisCore): EvmNftData => ({
     ...data,
     chain: EvmChain.create(data.chain, core),
-    contractType: validateValidEvmContractType(data.contractType),
+    contractType: noramlizeEvmNftContractType(data.contractType),
```
Comment (ErnoW) [inline packages/evmUtils/src/dataTypes/EvmNft/EvmNft.ts:56]: Could we not just something like:
```
    contractType: maybe(data.contractType, validateValidEvmContractType),
```

**Review by ErnoW — CHANGES_REQUESTED** [review]

File: packages/evmUtils/src/dataTypes/EvmNft/EvmNft.ts (hunk 1 above)
Comment (b4rtaz) [reply packages/evmUtils/src/dataTypes/EvmNft/EvmNft.ts:56]: In the previous version here's also a validation. I didn't change it.
https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/packages/evmUtils/src/dataTypes/EvmNftContractType.ts

Comment (b4rtaz) [reply packages/evmUtils/src/dataTypes/EvmNft/EvmNft.ts:56]: Maybe it's not needed.

Comment (b4rtaz) [reply packages/evmUtils/src/dataTypes/EvmNft/EvmNft.ts]: I've standardized the code.

```js
contractType: maybe(data.contractType, normalizeEvmNftContractType),
```

**Review by ErnoW — APPROVED** [review]

**Pull request merged**